packages/block-editor/src/components/iframe/index.js

```diff
@@ -168,8 +168,8 @@ function Iframe( { contentRef, children, head, tabIndex = 0, ...props }, ref ) {
 	const [ , forceRender ] = useReducer( () => ( {} ) );
 	const [ iframeDocument, setIframeDocument ] = useState();
 	const [ bodyClasses, setBodyClasses ] = useState( [] );
-	const styles = useParsedAssets( window.__editorAssets.styles );
-	const scripts = useParsedAssets( window.__editorAssets.scripts );
+	const styles = useParsedAssets( window.__editorAssets?.styles );
+	const scripts = useParsedAssets( window.__editorAssets?.scripts );
```

Comment:
I don't really understand what this is or how it relates to anything.

But hey, looks good, thank you for fixing it!

Reply:
I actually remember coming across this so I can verify it's a thing 😆 

Thanks for the fix @gwwar!